normalize/validate.go

```diff
@@ -359,4 +359,4 @@ func applyRecordTransforms(domain *models.DomainConfig) error {
 		}
 	}
 	return nil
-}
+}
```

Comment:
Looks like you are on windows? Line endings are a pain in the ass. I always have to set my editors and git config up to not touch line endings at all for my go projects.